plugins/org.obeonetwork.m2doc.doc/doc/user/M2Doc User Guide.html

```diff
@@ -173,6 +173,17 @@ <h4 id="Siriustableproviders">Sirius table providers</h4>
 			<b>Warning</b>: do not forget the single quotes ' around the string in an AQL Expression: 
 			<code>title:"'Table name'"</code>. The description ID does not need such quotes because it is of type String.
 		</p>
+		<h4 id="UpdatestrategyforSiriusRepresentation">Update strategy for Sirius Representation</h4>
+		<p>In both case Sirius diagram and Sirius table providers, the M2Doc generation can occur with a closed Sirius representation view. So the semantic and graphical models could be unsynchronized. A parameter in genConf model define the strategy to follow during M2Doc generation.</p>
```

Comment:
In the cases of generating a diagram with {m:diagram} or a table with {m:wtable}, the M2Doc generation can take place on an unsynchronized representation (diagram or table). Consequently, a parameter has been introduced in the genConf model to allow users to specify whether reprensentations must be refreshed before the M2Doc generation.